libs/@guardian/source-react-components-development-kitchen/src/footer-with-contents/FooterLinks.tsx

```diff
@@ -52,58 +64,78 @@ export interface FooterLinksProps {
 	/**
 	 * An array of links, specifying the link text and href, if the link is external, and any other HTML attributes for an anchor tag
 	 */
-	links?: FooterLink[];
+	links?: Array<FooterLink | FooterButton>;
 	/**
 	 * Force the links into a column layout below desktop, regardless of the amount of links
 	 */
 	forceColumns?: boolean;
 }
 
+const getLink = (link: FooterLink) => {
+	const { href, text, isExternal, ...linkAttrs } = link;
```

Comment:
Complete sideline, you could deconstruct in the arguments

```suggestion
const getLink = ({ href, text, isExternal, ...linkAttrs }: FooterLink) => {
```